.github/workflows/deploy-demo-preview.yml

```
@@ -0,0 +1,92 @@
+      # to link a PR to a list of environments later.
+      url: ${{ env.APP_URL }}
+    steps:
+    - uses: actions/checkout@master
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 3.0.x
+    - uses: actions/cache@v2
+      with:
+        path: demo/gemfiles/vendor/bundle
+        key: gems-build-kuby-main-ruby-3.0.x-${{ hashFiles('demo/gemfiles/kuby.gemfile.lock') }}
+    - name: Docker login
+      env:
+        AZURE_ACR_PASSWORD: ${{ secrets.AZURE_ACR_PASSWORD }}
+      run: echo $AZURE_ACR_PASSWORD | docker login primer.azurecr.io --username GitHubActions --password-stdin
+    - uses: Azure/login@v1
+      with:
+        creds: '{"clientId":"5ad1a188-b944-40eb-a2f8-cc683a6a65a0","clientSecret":"${{ secrets.AZURE_SPN_CLIENT_SECRET }}","subscriptionId":"550eb99d-d0c7-4651-a337-f53fa6520c4f","tenantId":"398a6654-997b-47e9-b12b-9515b896b4de"}'
+    - name: Bundle
+      run: |
+        gem install bundler -v '~> 2.3'
+        bundle install --jobs 4 --retry 3 --gemfile demo/gemfiles/kuby.gemfile --path vendor/bundle
+
+    - name: Get preview app info
+      run: ./.github/workflows/get-preview-app-info.sh
+
+    - name: 'Set env vars'
+      run: |
+        # Image tag is unique to each workflow run so that it always triggers a new deployment
+        echo "DOCKER_IMAGE=${{ env.IMAGE_URL }}:${{ env.COMMIT_REF }}-${{ github.run_number }}-${{ github.run_attempt }}" >> $GITHUB_ENV
+    - name: Pull latest
+      run: |
+        docker pull ${{ env.IMAGE_URL }}:latest
+        docker pull ${{ env.IMAGE_URL }}:latest-assets
+    - name: Build
+      env:
+        RAILS_MASTER_KEY: ${{ secrets.RAILS_MASTER_KEY }}
+      run: |
+        bin/kuby -e production build --only app -- -t $DOCKER_IMAGE
+        bin/kuby -e production build --only assets -- -t $DOCKER_IMAGE
+    - name: Push
+      run: bin/kuby -e production push
+    - name: Run ARM deploy
+      # This 'if' will be truth, if this workflow is...
+      #  - run as a workflow_dispatch
+      #  - run because of a push to main (or when added to a merge queue)
+      #  - run as a regular pull request
+      # But if it's a pull request, *and* for whatever reason, the pull
+      # request has "Auto-merge" enabled, don't bother.
+      # The idea is that if auto-merge has been abled, by humans or by
+      # bots, they have no intention of viewing the deployed preview anyway.
+      # This saves time because the PR can merge sooner.
+      if: ${{ !github.event.pull_request.auto_merge }}
+      uses: azure/arm-deploy@841b12551939c88af8f6df767c24c38a5620fd0d
+      with:
+        resourceGroupName: primer
+        subscriptionId: 550eb99d-d0c7-4651-a337-f53fa6520c4f
+        template: ./.github/workflows/deploy-demo-preview-template.json
+        deploymentName: ${{env.DEPLOYMENT_NAME}}
+        parameters: appName="${{env.APP_NAME}}"
+          containerImage="primer.azurecr.io/${{ env.IMAGE_REPO }}:latest"
```

I think this should be 
```suggestion
          containerImage="${{ DOCKER_IMAGE }}"
```